tests/ai_hub/agent_catalog/metadata/test_agent_catalog_template_artifacts.py

```
@@ -70,3 +72,33 @@ def test_agent_template_content_matches_agent_metadata(
+                errors.append(f"Agent '{name}' logo is empty")
+                continue
+            if not logo.startswith(prefix):
+                errors.append(f"Agent '{name}' logo is not a base64 SVG data URI: {logo[:50]!r}")
+                continue
+            try:
+                svg_bytes = base64.b64decode(s=logo.removeprefix(prefix))
+            except ValueError:
+                errors.append(f"Agent '{name}' logo base64 decoding failed")
+                continue
+            try:
+                ET.fromstring(text=svg_bytes.decode(encoding="utf-8"))
```

Ruff (0.16.1)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)